src/types.ts

```diff
@@ -66,3 +66,22 @@ export interface TaggedRecord {
 export interface UploadResponseBody {
   url: string;
 }
+
+enum PublicMapColumnType {
+  Boolean = "Boolean",
+  CommaSeparatedList = "CommaSeparatedList",
+  String = "String",
+}
```

Comment:
i copied it from generated types cause if I was importing I was getting a circular dependecy error - not sure why and how to solve it 🤔